tests/Imgix/Tests/UrlBuilderTest.php

```diff
@@ -302,10 +323,11 @@ public function testDprSrcsetWithDefaultQuality() {
 
     public function testDprSrcsetWithDisabledVariableQuality() {
```

Comment:
Maybe just a bad case of copy+paste, but I think we want to test that this srcset produces no `q` parameter when `$disableVariableQuality=true`.

Reply:
Yeah, there was a couple things up here. I revamped some things and briefly commented three tests that might cause confusion. I also found that I had not completely removed a function parameter (a hold over from removing custom qualities).